xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -229,13 +241,19 @@
         <Parameter Name="numberOfCharactersWritten" Type="System.Int32&amp;" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="unicodeScalar">To be added.</param>
-        <param name="buffer">To be added.</param>
-        <param name="bufferLength">To be added.</param>
-        <param name="numberOfCharactersWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unicodeScalar">A Unicode scalar value.</param>
+        <param name="buffer">A pointer to the buffer to which to write the encoded text.</param>
+        <param name="bufferLength">The length of the destination <paramref name="buffer"/> in characters.</param>
+        <param name="numberOfCharactersWritten">The number of characters written to the <paramref name="buffer"/>.</param>
```

Comment:
Since this is an out param, change this to, "When the method returns, indicates the number of characters..."